metrics/src/main/java/com/google/gson/metrics/BagOfPrimitives.java

```diff
@@ -43,14 +43,12 @@ public int getIntValue() {
   }
 
   public String getExpectedJson() {
-    StringBuilder sb = new StringBuilder();
-    sb.append("{");
-    sb.append("\"longValue\":").append(longValue).append(",");
-    sb.append("\"intValue\":").append(intValue).append(",");
-    sb.append("\"booleanValue\":").append(booleanValue).append(",");
-    sb.append("\"stringValue\":\"").append(stringValue).append("\"");
-    sb.append("}");
-    return sb.toString();
+    return "{" +
+            "\"longValue\":" + longValue + "," +
```

Comment:
Not sure if the indentation here is correct, I think this should have 4 spaces relative from the start of `return` (instead of 8).

Reply:
Yes. Google Java Style (which admittedly is not always followed throughout the Gson code) suggests 4-space continuation lines, and also moving `+` from the end of each line to the start of the following line.